### DISCUSSION TITLE
Character AI not linking unless it's a random bot?

### DISCUSSION THREAD

**Discussion by classic-DSW** [drizzle-mizzle/CharacterAI-Discord-Bot#140] [Q&A]
When I set a character, no matter which character, i keep getting the same "500 server error" message, but it works perfectly fine when I link characters using the ".ai # find <insert proxy here> command 
![erorr](https://github.com/drizzle-mizzle/CharacterAI-Discord-Bot/assets/128752380/e35730dc-272e-4b91-b2d5-0c92c7302330)
^ error message that comes up in terminal 
![help me](https://github.com/drizzle-mizzle/CharacterAI-Discord-Bot/assets/128752380/d73eb1fa-836b-47d3-bb4e-f0d7b3d4a2cc)
^ this comes up, too
![genshit unimpactful](https://github.com/drizzle-mizzle/CharacterAI-Discord-Bot/assets/128752380/be711d99-0058-48de-ac8f-30bfba57b0f7)
^ but it works when i set a random character by using the # find tool

**Comment by drizzle-mizzle** [comment]
I think your character ID is either wrong or private

**Reply by drizzle-mizzle** [reply]
Well, second is not likely to be a problem, especially if it's your own character. Then it's first, and I can't check it unless you show me the ID.

**Reply by classic-DSW** [reply]
98nYIiB_HPecdK3icnj8SLeTrxhd1D6ytUNMsXsyNI4 
this is the character id, it also doesn't work when i choose a random public character id and put it in

**Reply by classic-DSW** [reply]
https://beta.character.ai/chat?char=98nYIiB_HPecdK3icnj8SLeTrxhd1D6ytUNMsXsyNI4 heres a link to the character as well

**Reply by drizzle-mizzle** [reply]
![image](https://github.com/drizzle-mizzle/CharacterAI-Discord-Bot/assets/55811932/6b6815fa-285a-41ff-aadf-27467b82f2dc)

**Reply by drizzle-mizzle** [reply]
If you would not bedaub your screenshot with that red line, I would noticed right away that you put the id in `<>` for some reason. Ofc you don't need to do that.

**Reply by drizzle-mizzle** [reply]
![image](https://github.com/drizzle-mizzle/CharacterAI-Discord-Bot/assets/55811932/e5676500-7fcc-447a-80cb-9dc5fa0f17a1)
